austation/code/modules/antagonists/bloodsucker/bloodsucker_life.dm

```diff
@@ -0,0 +1,536 @@
+			target.death(0)
+
+	///////////
+	// Shift Body Temp (toward Target's temp, by volume taken)
+	owner.current.bodytemperature = ((owner.current.blood_volume * owner.current.bodytemperature) + (blood_taken * target.bodytemperature)) / (owner.current.blood_volume + blood_taken)
+	// our volume * temp, + their volume * temp, / total volume
+	///////////
+
+	// Reduce Value Quantity
+	if (target.stat == DEAD)	// Penalty for Dead Blood
+		blood_taken /= 3
+	if (!ishuman(target))		// Penalty for Non-Human Blood
+		blood_taken /= 2
+	//if (!iscarbon(target))	// Penalty for Animals (they're junk food)
+
+
+	// Apply to Volume
+	AddBloodVolume(blood_taken)
+
+	// Reagents (NOT Blood!)
+	if(target.reagents && target.reagents.total_volume)
+		//target.reagents.reaction(owner.current, INGEST, 1 / target.reagents.total_volume) // Run Reaction: what happens when what they have mixes with what I have?	DEAD CODE MUST REWORK
+		target.reagents.trans_to(owner.current, 1)	// Run transfer of 1 unit of reagent from them to me.
+
+	// Blood Gulp Sound
+	owner.current.playsound_local(null, 'sound/effects/singlebeat.ogg', 40, 1) // Play THIS sound for user only. The "null" is where turf would go if a location was needed. Null puts it right in their head.
+
+
+/datum/mood_event/drankblood
+	description = "<span class='nicegreen'>I have fed greedly from that which nourishes me.</span>\n"
+	mood_change = 10
+	timeout = 900
+/datum/mood_event/drankblood_bad
+	description = "<span class='boldwarning'>I drank the blood of a lesser creature. Disgusting.</span>\n"
+	mood_change = -4
+	timeout = 900
+/datum/mood_event/drankblood_dead
+	description = "<span class='boldwarning'>I drank dead blood. I am better than this.</span>\n"
+	mood_change = -7
+	timeout = 900
+/datum/mood_event/drankblood_synth
+	description = "<span class='boldwarning'>I drank synthetic blood. What is wrong with me?</span>\n"
+	mood_change = -7
+	timeout = 900
+/datum/mood_event/drankkilled
+	description = "<span class='boldwarning'>I drank from my victim until they died. I feel...less human.</span>\n"
+	mood_change = -12
+	timeout = 6000
+/datum/mood_event/madevamp
+	description = "<span class='boldwarning'>A soul has been cursed to undeath by my own hand.</span>\n"
+	mood_change = -10
+	timeout = 10000
+/datum/mood_event/vampatefood
+	description = "<span class='boldwarning'>Mortal nourishment no longer sustains me. I feel unwell.</span>\n"
+	mood_change = -6
+	timeout = 1000
+/datum/mood_event/coffinsleep
+	description = "<span class='nicegreen'>I slept in a coffin during the day. I feel whole again.</span>\n"
+	mood_change = 8
+	timeout = 1200
+/datum/mood_event/daylight_1
+	description = "<span class='boldwarning'>I slept poorly in a makeshift coffin during the day.</span>\n"
+	mood_change = -3
+	timeout = 1000
+/datum/mood_event/daylight_2
+	description = "<span class='boldwarning'>I have been scorched by the unforgiving rays of the sun.</span>\n"
+	mood_change = -6
+	timeout = 1200
+
```

Comment:
```suggestion
/datum/mood_event/wellfed_blood
	description = "<span class='nicegreen'>I'm saturated with blood, and it feels so good.</span>\n"
	mood_change = 8
/datum/mood_event/fed_blood
	description = "<span class='nicegreen'>I have enough blood.</span>\n"
	mood_change = 5

```

Please review description I am horrible at writing descriptions

Reply:
ahaha this looks like it could work. Those desriptions are kek enough but I can try and look into it